corecfg/proxy.go

```diff
@@ -71,3 +74,25 @@ func handleProxyConfiguration(tr Conf) error {
 
 	return nil
 }
+
+func handleProxyStore(tr Conf) error {
+	proxyStore, err := coreCfg(tr, "proxy.store")
+	if err != nil {
+		return err
+	}
+
+	if proxyStore != "" {
```

Comment:
(nitpick) I personally would make this `if proxyStore == nil { return nil }`

Reply:
how can I do that? proxyStore is a string, and "" is also the only way to unset config atm

Reply:
Ups, sorry, silly me. I should have written: `if proxyStore != "" { return nil }`, i.e. just to return early not to change any logic.

Reply:
ah, the nil confused me